net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardServer.xml

```diff
@@ -0,0 +1,53 @@
+<form>
+    <field>
+        <id>server.enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help>This will enable or disable the server config.</help>
+    </field>
+    <field>
+        <id>server.name</id>
+        <label>Name</label>
+        <type>text</type>
+        <help>Set the name for this instance.</help>
+    </field>
+    <field>
+        <id>server.instance</id>
+        <label>Instance</label>
+        <type>info</type>
+        <help>Set the instance number needed for interface calculation. It has to be unique for each instance.</help>
+    </field>
+    <field>
+        <id>server.pubkey</id>
+        <label>Public Key</label>
+        <type>text</type>
+        <help>Public key of this instance. After saving this field is automatically and you can give out the key to other parties.</help>
```

Comment:
automatically -> automatic

This field should be marked as read-only.

Reply:
Fixed